core/src/test/scala/cell/base.scala

```diff
@@ -1163,9 +1163,9 @@ class BaseSuite extends FunSuite {
   test("putFinal: result passed to callbacks") {
     val latch = new CountDownLatch(1)
 
-    implicit val setLattice = new Lattice[Set[Int]] {
-      def join(curr: Set[Int], next: Set[Int]) = curr ++ next
-      def empty = Set.empty[Int]
+    implicit val setUpdater = new AggregationUpdater[Set[Int]] {
+      def join(curr: Set[Int], v2: Set[Int]): Set[Int] = curr ++ v2
+      def bottom = Set.empty[Int]
```

Comment:
This should definitely remain a lattice (because it is!), and instead the implicit conversion should convert the lattice to the required updater.